test/test_align_cli.py

```diff
@@ -126,6 +126,21 @@ def test_invoke_align(self):
             "image-for-page1.jpg was not on disk, cannot have been copied",
         )
 
+        results_html = self.runner.invoke(
+            align,
+            [
+                join(self.data_dir, "ej-fra-package.xml"),
+                join(self.data_dir, "ej-fra.m4a"),
+                join(output, "html"),
+                "--html",
+            ],
+        )
+        self.assertEqual(results_html.exit_code, 0)
+        self.assertTrue(
+            exists(join(output, "html", "html.html")),
+            "succesful html alignment should have created output.html",
+        )
+
```

Comment:
Hum, this is pretty weak validation... Maybe inspect the contents and make sure it contains the strings `read-along text="data:application/xml;base64`, `alignment="data:application/smil+xml;base64` and `audio="data:audio/mp4;base64`?

Also, the message in case of error should say "should have created html/html.html", rather than "output.html".